scripts/docker_clang_env.sh

```diff
@@ -12,6 +12,8 @@ if [ "$SANITIZER" == "undefined_sanitizer" ]
   then FLAGS="-fsanitize=address -fsanitize=undefined -fsanitize-blacklist=/scratch/source/trilinos/release/DataTransferKit/scripts/undefined_blacklist.txt"
   # Suppress leak sanitizer on functions we have no control of.
   export LSAN_OPTIONS=suppressions=/scratch/source/trilinos/release/DataTransferKit/scripts/leak_blacklist.txt
+  # Set the symbolizer so we get line outputs from errors for debugging.
```

Comment:
What the h does "[symbolizing the reports"](https://clang.llvm.org/docs/AddressSanitizer.html#symbolizing-the-reports)" means?